datasets/mnist/pipelines/mnist/pipeline.yaml

```diff
@@ -0,0 +1,118 @@
+          request_memory: "2G"
+          request_cpu: "200m"
+          request_ephemeral_storage: "8G"
+
+    - operator: "KubernetesPodOperator"
+      description: "Task to copy `train-images-idx3-ubyte.gz` from MNIST  Database to GCS"
+      args:
+        task_id: "download_and_process_source_zip_file_2"
+        name: "mnist"
+        namespace: "composer"
+        service_account_name: "datasets"
+        image_pull_policy: "Always"
+        image: "{{ var.json.mnist.container_registry.run_csv_transform_kub }}"
+        env_vars:
+          SOURCE_URL: "http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz"
+          SOURCE_FILE: "files/train-images-idx3-ubyte.gz"
+          TARGET_FILE: "files/train-images-idx3-ubyte.gz"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/mnist/mnist/train-images-idx3-ubyte.gz"
+          PIPELINE_NAME: "mnist"
+        resources:
+          request_memory: "2G"
+          request_cpu: "200m"
+          request_ephemeral_storage: "8G"
+
+    - operator: "KubernetesPodOperator"
+      description: "Task to copy `train-labels-idx1-ubyte.gz` from MNIST  Database to GCS"
+      args:
+        task_id: "download_and_process_source_zip_file_3"
+        name: "mnist"
+        namespace: "composer"
+        service_account_name: "datasets"
+        image_pull_policy: "Always"
+        image: "{{ var.json.mnist.container_registry.run_csv_transform_kub }}"
+        env_vars:
+          SOURCE_URL: "http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz"
+          SOURCE_FILE: "files/train-labels-idx1-ubyte.gz"
+          TARGET_FILE: "files/train-labels-idx1-ubyte.gz"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/mnist/mnist/train-labels-idx1-ubyte.gz"
+          PIPELINE_NAME: "mnist"
+        resources:
+          request_memory: "2G"
+          request_cpu: "200m"
+          request_ephemeral_storage: "8G"
+
+    - operator: "KubernetesPodOperator"
+      description: "Task to copy `t10k-labels-idx1-ubyte.gz` from MNIST  Database to GCS"
+      args:
+        task_id: "download_and_process_source_zip_file_4"
+        name: "mnist"
+        namespace: "composer"
+        service_account_name: "datasets"
+        image_pull_policy: "Always"
+        image: "{{ var.json.mnist.container_registry.run_csv_transform_kub }}"
+        env_vars:
+          SOURCE_URL: "http://yann.lecun.com/exdb/mnist/t10k-labels-idx1-ubyte.gz"
+          SOURCE_FILE: "files/t10k-labels-idx1-ubyte.gz"
+          TARGET_FILE: "files/t10k-labels-idx1-ubyte.gz"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/mnist/mnist/t10k-labels-idx1-ubyte.gz"
+          PIPELINE_NAME: "mnist"
+        resources:
+          request_memory: "2G"
+          request_cpu: "200m"
+          request_ephemeral_storage: "8G"
+
+  graph_paths:
+    - "download_and_process_source_zip_file"
```

Comment:
with this you are only executing the "download_and_process_source_zip_file" pipeline, but there are 3 other pipelines here that are not being executed.  Either include the other pipelines or remove them from the pipeline.yaml file.

Reply:
included, done